Automattic/jetpack
# Add docker setup to run e2e tests locally

This configuration is mostly based on Gutenberg's docker setup, and it reuses most of its functionality.

#### Changes proposed in this Pull Request:

* Adds a docker configuration for running e2e tests locally 

#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
* p9dueE-Ju-p2

#### Testing instructions:






* make sure that docker is up and running
* run `./tests/e2e/bin/docker-e2e-cli.sh setup`.
* wait until the command is done.
* open the site link which was printed out from above command: `Open ${WP_SITE_URL} to see your site!` 
* check the site, try log into wp-admin (creds: wordpress/wordpress), make sure that Jetpack is active
* run the tests: `PUPPETEER_HEADLESS=false yarn test-e2e --verbose --runInBand`

#### Proposed changelog entry for your changes:

* n/a
